data/json/items/tool_armor.json

```diff
@@ -2515,5 +2515,53 @@
         "material_thickness": 1,
         "use_action": "SOLARPACK_OFF",
         "flags": [ "FRAGILE", "OUTER", "ONLY_ONE" ]
+    },
+    {
+        "id": "helmet_riot",
+        "type": "TOOL_ARMOR",
+        "category": "armor",
+        "name": "riot helmet",
+        "description": "A helmet with a plastic shield that covers your entire face.  Activate to raise the face shield.",
+        "use_action": { "type": "transform", "target": "helmet_riot_raised", "msg": "You raise your visor." },
+        "weight": 2260,
+        "volume": 9,
+        "price": 22000,
+        "to_hit": -1,
+        "bashing": 8,
+        "material": [ "plastic", "steel" ],
+        "symbol": "[",
+        "color": "dark_gray",
+        "covers": [ "HEAD", "EYES", "MOUTH" ],
+        "coverage": 95,
+        "encumbrance": 20,
+        "warmth": 10,
+        "material_thickness": 3,
+        "environmental_protection": 2,
+        "techniques": [ "WBLOCK_1" ],
+        "flags": [ "WATERPROOF", "STURDY" ]
+    },
+    {
+        "id": "helmet_riot_raised",
+        "type": "TOOL_ARMOR",
+        "category": "armor",
+        "name": "riot helmet(raised visor)",
```

Comment:
There should be a space after "helmet". Also needs a "name_plural" entry (see other item definitions in this file) as its current plural form is "riot helmet(raised visor)s".

Reply:
k, will fix